environment.yml

```diff
@@ -4,3 +4,8 @@ channels:
   - defaults
 dependencies:
   - gdal
+  - fiona
+  - boto3
+  - pip
```

Comment:
I'm guessing we don't need `pip` as a dependency here? If anything, it'd be a build dependency, not a runtime dependency.

Reply:
I've accepted your modifications but building failed for me without the pip dependency and pip-sourced stactools==0.2.1a1.

Reply:
Are you working with the latest pip? What are the errors? I haven't seen this before.

Reply:
```
ResolvePackageNotFound: 
  - stactools==0.2.1a1
```

I assume it's the latest pip as it's inside the docker container created via `cibuild`

Reply:
Oh I'm sorry! I thought we were editing `setup.cfg`, but I see here that this is the conda environment file. Can you move all the dependencies except for GDAL to the `setup.cfg` file and things should go.

Our conda vs. pip story is a bit confused, for which I apologize. Basically we use conda only for the GDAL binary package, and use pip for everything else.

Reply:
No problem! Thanks for helping to resolve the issues so promptly. I've made the changes you suggested. `cibuild` runs fine now.